internal/app/machined/pkg/runtime/v1alpha1/platform/vultr/vultr.go

```diff
@@ -151,14 +183,7 @@ func (v *Vultr) NetworkConfiguration(ctx context.Context, _ state.State, ch chan
 		return err
 	}
 
-	extIP, err := download.Download(ctx, VultrExternalIPEndpoint,
```

Comment:
does Vultr have external IPs always available in network config and attached to the node (vs. like AWS/GCP when public IP is not on the node)?